File: src/tool/spring-boot-generator/output/spider/db/taskInfo.js
let taskInfo = {
	name: "taskInfo",
	description: "任务信息",
	columns: {
		id: {
			name: "id",
			description: "编号",
			isPk: true,
			autoIncrement: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		userId: {
			name: "userId",
			description: "用户id",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		mainConfigId: {
			name: "mainConfigId",
			description: "主配置id",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		description: {
			name: "description",
			description: "描述",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		level: {
			name: "level",
			description: "任务等级",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		status: {
			name: "status",
			description: "任务状态",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		queueId: {
			name: "queueId",
			description: "queue id",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		topicId: {
			name: "topicId",
			description: "topic id",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		createTime: {
			name: "createTime",
			description: "创建时间",
			nullable: true,
			type: {
				name: "bigint",
				length: null
			}
		},
		endTime: {
			name: "endTime",
			description: "结束时间",
			nullable: true,
			type: {
				name: "bigint",
				length: null
			}
		}
	}
};

let columnsArray = [
	"id",
	"userId",
	"mainConfigId",
	"description",
	"level",
	"status",
	"queueId",
	"topicId",
	"createTime",
	"endTime"
];

function create() {
  return {};
}

taskInfo.columnsArray=columnsArray;
taskInfo.create=create;
module.exports=taskInfo;

File: src/tool/spring-boot-generator/output/spider/spider/src/main/java/com/spider/pojo/entity/TaskInfo.java
package com.spider.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;



@ApiModel("任务信息")
@Getter
@Setter
public class TaskInfo  {

    @ApiModelProperty("编号")
    private Integer id;

    @ApiModelProperty("用户id")
    private Integer userId;

    @ApiModelProperty("主配置id")
    private Integer mainConfigId;

    @ApiModelProperty("描述")
    private String description;

    @ApiModelProperty("任务等级")
    private Integer level;

    @ApiModelProperty("任务状态")
    private Integer status;

    @ApiModelProperty("queue id")
    private String queueId;

    @ApiModelProperty("topic id")
    private Integer topicId;

    @ApiModelProperty("创建时间")
    private Integer createTime;

    @ApiModelProperty("结束时间")
    private Integer endTime;

}